### PR TITLE
feat : add linode storage adapter

### PR DESCRIPTION
**Pull request by everly-gif**
## What does this PR do?

This PR will add support for Linode Object Storage.

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Unreleased Version
 - Renamed `providers` to `authProviders` in project collection **Breaking Change**
+- Support for Backblaze adapter in Storage
+- Support for Linode adapter in Storage
```
Comment (Meldiron) [inline CHANGES.md:4]: This should go into 0.14.2 section probably. Let's sync with master branch, and update it.

File: app/config/variables.php [hunk 2]
```diff
@@ -510,6 +510,70 @@
                 'required' => false,
                 'question' => '',
             ],
+            [
+                'name' => '_APP_STORAGE_BACKBLAZE_ACCESS_KEY',
+                'description' => 'Backblaze access key. Required when the storage adapter is set to Backblaze. Your Backblaze keyID will be your access key. You can get your keyID from your Backblaze console.',
+                'introduction' => '0.13.0',
```
Comment (Meldiron) [inline app/config/variables.php:516]:
```suggestion
                'introduction' => '0.14.0',
```

Maybe even 0.14.2? Not sure

Comment (Meldiron) [reply app/config/variables.php:516]: Let's update this for all new vars.

Comment (everly-gif) [reply app/config/variables.php]: I have addressed all of these in the latest PR, which has been merged now by torsten. So closing this PR. Thanks for the review tho😊

**Pull request closed without merging**